src/main/java/net/minestom/server/event/player/AsyncPlayerPreLoginEvent.java

```diff
@@ -16,11 +17,14 @@
 public class AsyncPlayerPreLoginEvent implements Event {
 
     private GameProfile gameProfile;
+    private final PlayerConnection connection;
```

Comment:
nitpick, but connection should be ordered before gameprofile

Reply:
I don't understand why, but okay